### ISSUE TITLE
[Help / question] Make widget seperate on docklike mod

### ISSUE THREAD

**Issue by daflame899** [ramensoftware/windows-11-taskbar-styling-guide#145]
Is there a way to put this widget in the left corner (seperating it from the taskbar)?

![Capture d’écran 2024-12-07 103022](https://github.com/user-attachments/assets/f2266505-fef0-42c8-a269-0c657f6cd644)

**Comment by bbmaster123** [comment]
yes it can be done, but its not perfect. I recommend reading through [this thread](https://github.com/ramensoftware/windows-11-taskbar-styling-guide/issues/71) where I had helped another user to do this.

**Comment by daflame899** [comment]
> yes it can be done, but its not perfect. I recommend reading through [this thread](https://github.com/ramensoftware/windows-11-taskbar-styling-guide/issues/71) where I had helped another user to do this.

I pasted the code from there in advanced mod settings but it didn't work, I'm not sure if I did it wrong but what I've tried doesn't seem to work

**Comment by bbmaster123** [comment]
> I'm not sure if I did it wrong

I don't know either, its hard to help without knowing what you've tried! :)

It should be as simple as copying the whole code from [here](https://github.com/ramensoftware/windows-11-taskbar-styling-guide/issues/71#issuecomment-2286776821) and replacing everything already in the mod settings box. There shouldn't be any json error messages when you click save. 

If you need help combining what you have already with the new code, share what you have here and I'll be happy to help with that :)

**Comment by bbmaster123** [comment]
I ran out of time, but I can come back to this later

Here's what I have for you so far, you might need to adjust some margins

```
{
  "theme": "",
  "controlStyles[0].target": "Grid#RootGrid > Taskbar.TaskbarBackground > Grid",
  "controlStyles[0].styles[0]": "Margin=800,4,800,4",
  "controlStyles[0].styles[1]": "CornerRadius=14",
  "controlStyles[0].styles[2]": "Background:=<AcrylicBrush TintColor=\"Transparent\" TintOpacity=\"0.1\" Opacity=\"1\" TintLuminosityOpacity=\"0.05\"/>",
  "controlStyles[1].target": "Grid#SystemTrayFrameGrid",
  "controlStyles[1].styles[0]": "Margin=0,4,4,4",
  "controlStyles[1].styles[1]": "CornerRadius=14",
  "controlStyles[1].styles[2]": "Background:=",
  "controlStyles[1].styles[3]": "BorderBrush=#4C5887",
  "controlStyles[1].styles[4]": "BorderThickness=1",
  "controlStyles[2].target": "Taskbar.AugmentedEntryPointButton#AugmentedEntryPointButton > Taskbar.TaskListButtonPanel#ExperienceToggleButtonRootPanel > Border#BackgroundElement",
  "controlStyles[2].styles[0]": "Background:=<AcrylicBrush TintColor=\"Transparent\" TintOpacity=\"0.1\" Opacity=\"1\" TintLuminosityOpacity=\"0.05\" FallbackColor=\"{ThemeResource SystemChromeLowColor}\" />",
  "controlStyles[0].styles[3]": "BorderBrush=#8490BD",
  "controlStyles[0].styles[4]": "BorderThickness=1"
}
```
On the bright side, I fixed the issue of the widgets button moving around when opening closing windows

![image](https://github.com/user-attachments/assets/fa739f2b-11f6-4ce2-a44c-612f56f1eef4)
